.gitignore

```diff
@@ -9,3 +9,7 @@
 
 # rspec failure tracking
 .rspec_status
+
+.byebug_history
+
+Gemfile.lock
```

Comment:
According to bundler, the `Gemfile.lock` should be committed, see https://bundler.io/guides/faq.html#using-gemfiles-inside-gems